### PR TITLE
#163722901 Secured all API endpoints with JWT

### PR DESCRIPTION
**Pull request by kenzdozz**
#### What does this PR do?
Secured all API endpoints with JWT
#### Description of Task to be completed?
- secured all endpoints with jwt
- updated test files to test secured endpoints
- ensured that all tests are passing
#### How should this be manually tested?
Run `npm test` - All test passes
#### What are the relevant pivotal tracker stories?
[163684585](https://www.pivotaltracker.com/story/show/163722901/comments/199198877)

### CODE REVIEW

File: controllers/Auth.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      const user = await User.where(['email', '=', email]).first();
+
+      if (!user) {
+        return Response.send(res, codes.notFound, {
+          error: 'User does not exist.',
+        });
+      }
+
+      const emailtoken = Math.random().toString(15).substring(2);
+      const transporter = nodemailer.createTransport({
```
Comment (veeqtor) [inline controllers/Auth.js]: You can abstract this into a separate file.

File: controllers/Auth.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        host: 'smtp.ethereal.email',
+        port: 587,
+        auth: {
+          user: 'oqmeslufjugyffgd@ethereal.email',
+          pass: 'kjwYFsNjzTWMSYCRmh',
+        },
+      });
+      const mailOptions = {
```
Comment (veeqtor) [inline controllers/Auth.js]: same with this.

File: controllers/Auth.js (hunk 1 above)
Comment (kenzdozz) [reply controllers/Auth.js]: Okay, I will do just that

File: controllers/Auth.js (hunk 2 above)
Comment (kenzdozz) [reply controllers/Auth.js]: Noted

**Pull request merged**